docs/hugo/content/introduction/metrics.md

```diff
@@ -0,0 +1,45 @@
+# ASOv2 Metrics
+
+We have a set of prometheus metrics exposed for ASOv2 which can be helpful in diagnosability and tracing. 
+The metrics exposed, fall into two groups of `Azure based metrics` and `reconciler metrics(controller-runtime)` 
+combined.
+
+## Enabling the metrics
+
+By default, metrics scraping is turned on for ASOv2, which can be turned off using helm chart or Deployment yaml.
```

Comment:
```suggestion
By default, metrics scraping is turned on for ASOv2. It can be turned off using the Helm chart <What parameter?> or by editing the Deployment yaml <what parameter>?
```